Directory.Packages.props

```diff
@@ -66,7 +66,7 @@
     <PackageVersion Include="Vanara.PInvoke.User32" Version="3.4.11" />
     <PackageVersion Include="Vanara.PInvoke.Shell32" Version="3.4.11" />
     <PackageVersion Include="WinUIEx" Version="2.2.0" />
-    <PackageVersion Include="WPF-UI" Version="3.0.0-preview.4" />
+    <PackageVersion Include="WPF-UI" Version="2.1.0" />
```

Comment:
Any reason for the package downgrade?

Reply:
Good catch I'll revert back to `3.0.0-preview.4`